wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/episode/EpisodeScreen.kt

```diff
@@ -0,0 +1,222 @@
+package au.com.shiftyjelly.pocketcasts.wear.ui.episode
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight.Companion.W700
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.wear.compose.foundation.lazy.items
+import au.com.shiftyjelly.pocketcasts.compose.components.TextP50
+import au.com.shiftyjelly.pocketcasts.models.type.EpisodePlayingStatus
+import au.com.shiftyjelly.pocketcasts.models.type.EpisodeStatusEnum
+import au.com.shiftyjelly.pocketcasts.podcasts.view.episode.DownloadButtonState
+import au.com.shiftyjelly.pocketcasts.utils.Util
+import au.com.shiftyjelly.pocketcasts.wear.ui.component.WatchListChip
+import com.google.android.horologist.compose.layout.ScalingLazyColumn
+import com.google.android.horologist.compose.layout.ScalingLazyColumnState
+import au.com.shiftyjelly.pocketcasts.images.R as IR
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+
+@Composable
+fun EpisodeScreen(
+    columnState: ScalingLazyColumnState,
+    navigateToPodcast: (podcastUuid: String) -> Unit,
+    navigateToUpNextOptions: () -> Unit,
+    navigateToConfirmDeleteDownload: () -> Unit,
+    navigateToRemoveFromUpNextNotification: () -> Unit,
+) {
+
+    val viewModel = hiltViewModel<EpisodeViewModel>()
+    val state = viewModel.stateFlow.collectAsState().value
+    if (state !is EpisodeViewModel.State.Loaded) return
+
+    val episode = state.episode
+    val podcast = state.podcast
+    val context = LocalContext.current
+
+    ScalingLazyColumn(
+        columnState = columnState,
+        modifier = Modifier
+            .fillMaxSize()
+    ) {
+
+        val headingLineHeight = 14.sp
+        item {
+            TextP50(
+                text = episode.title,
+                maxLines = 2,
+                fontWeight = W700,
+                lineHeight = headingLineHeight,
+                textAlign = TextAlign.Center,
+                modifier = Modifier.padding(horizontal = 8.dp)
+            )
+        }
+
+        item {
+            TextP50(
+                text = podcast.title,
+                maxLines = 1,
+                color = Color(0xFFDADCE0),
```

Comment:
I think I need to create a watch theme [similar to what we have for the automotive app](https://github.com/Automattic/pocket-casts-android/blob/d1502d9bf89fec3e19098f3ec78bc41e1473b447/modules/features/cartheme/src/main/res/values/colors.xml) (but with compose in mind) to handle these colors. I'll do that in a separate PR though.